.environment/gitleaks/gitleaks-config.toml

```diff
@@ -12,6 +12,7 @@ title = "PRIME ReportStream Gitleaks Configuration"
         'yarn\.lock$',
     ]
     paths = [
+        '.environment/sftp-conf',
```

Comment:
@jeremy-page this PR will probably be merged before yours so you may need to verify the exemption is still good (it seems straightforward, just want to double-check)